src/current/cockroachcloud/custom-metric-chart-page.md

```diff
@@ -0,0 +1,46 @@
+---
+title: Custom Metric Chart Page
+summary: The Custom Metric Chart page lets you create one or multiple custom charts showing the time series data for an available metric or combination of metrics.
+toc: true
+---
+
+The **Custom Metric Chart** page is available for CockroachDB {{ site.data.products.serverless }} clusters. To view this page, select a cluster from the [**Clusters** page]({% link cockroachcloud/cluster-management.md %}#view-clusters-page), and click **Metrics** in the **Monitoring** section of the left side navigation. Navigate to the **Custom** tab to create one or multiple custom charts showing the time series data for an available metric or combination of metrics.
+
+{{site.data.alerts.callout_info}}
+{% include_cached feature-phases/preview.md %}
+{{site.data.alerts.end}}
+
+## Use the Custom Metric Chart page
+
+<img src="{{ 'images/cockroachcloud/custom-metric-chart.png' | relative_url }}" alt="Custom Metric Chart" style="border:1px solid #eee;max-width:100%" />
+
+On the **Custom Metric Chart** page, you can set the time span for all charts, add new custom charts, and customize each chart:
+
+- To set the time span for the page, use the time interval selector at the top of the tab to filter the view for a predefined or custom time interval. Use the navigation buttons to move to the previous, next, or current time interval. When you select a time interval, the same interval is selected for all charts on all tabs of the [Metrics page]({% link cockroachcloud/metrics-page.md %}).
+- To add a chart, click **Create** to create the first custom chart or **Add Chart** to create subsequent custom charts. The [**Create custom chart** modal](#create-custom-chart-modal) is displayed.
+- To edit a chart, click the pencil icon to display the [**Edit custom chart** modal](#create-custom-chart-modal).
+- To delete a chart, click the trash icon.
+
+## Create custom chart modal
+
+In the **Create custom chart** modal, you can customize each chart.
+
+<img src="{{ 'images/cockroachcloud/custom-metric-chart-create.png' | relative_url }}" alt="Create Custom Chart" style="border:1px solid #eee;max-width:70%" />
+
+- Under **Select metrics**, add the metrics to be queried, and how they'll be combined and displayed. Options include:
+{% include cockroachcloud/ui-custom-metric-chart-page.html %}
+- Under **Provide a chart name**, enter title text or keep default title with names of selected metrics.
+- Under **Preview**, choose a time span that contains data for the selected metrics.
+- Once the chart is verified in the **Preview**, click **Submit** to add the new custom chart or click **Close** to return to the existing **Custom Metric Chart** page.
```

Comment:
What are the default values of the Preview? We should call it out here. @THardy98

Reply:
The only default in the Preview is that it uses the page's selected time interval. Updated with this information.